doc/nrf/doc_styleguide.rst

```diff
@@ -120,6 +208,41 @@ To link directly to a doxygen reference from RST, use the following Breathe doma
 
       :cpp:enumerator:`ENUM_VALUE <DOXYGEN_GROUP::ENUM_VALUE>`
 
+PCB names
+=========
+
+When referring to specific Printed Circuit Board elements, use capitalization.
+For example, **Button 1**, **Switch 3**, **LED 1**.
```

Comment:
So would we write "Press Button 1 to..." or "Press **Button 1** to..."?
No big preference here (slight tendency for no bold).

Reply:
Hm, this is a bit more complicated. Rephrased in the latest version.